leaf-ui/js/onFunctions.js

```diff
@@ -530,21 +530,25 @@ paper.on("link:options", function (cell) {
 
         // If single path has been run backend analysis
         if (singlePathRun === true) {
-            $("body").addClass("spinning"); // Adds spinner animation to page
             var curResult = curRequest.previousAttributes().results.findWhere({ selected: true });
             curRequest.set('action', 'allNextStates');
             curRequest.set('previousAnalysis', curResult);
 
-            // If the last time point is selected, error message shows that you can't open Next State
-            if ((curResult.get('timePointPath').length - 1) === curResult.get('selectedTimePoint')) {
-                swal("Error: Cannot explore next states with last time point selected.", "", "error");
-                $("body").removeClass("spinning"); // Remove spinner from page
-            } else {
-                backendSimulationRequest(curRequest);
+            if (EVO.sliderOption == '1' || EVO.sliderOption == '2') {
+                swal("Error: Cannot explore next states from percent or time EVO options.", "", "error");
```

Comment:
Not sure if you still want this message when Evo is turned on but the user hasn't clicked a Result yet

![image](https://user-images.githubusercontent.com/47505178/144724110-35090fee-96e1-4c2d-9f13-fa34010a99d4.png)